### PR TITLE
Doc/readme-updates

### PR DESCRIPTION
**Pull request by ttybitnik**
Hey there @masasam,

While revisiting the package and Hugo documentation to find a way to implement featured post images, I came up with some `README` modifications listed at the end.

Rather than introducing an images field into the org template, as we did for categories in #73, which was a good default, I wrote a brief explanation on how to include custom front matter fields/keywords for `easy-hugo-org-header` users, based on Front Matter [docs](https://gohugo.io/content-management/front-matter/). I think this approach makes more sense this time since the images field wouldn't be a sane default for the template, due its narrow use case and its dissonance from other templates fields in the repo. 

The `easy-hugo-list-draft` is working fine with the `easy-hugo-org-header`  set, so I removed a statement on that. Optionally, I also did some changes in the `intro` of the README, centering the logo and restructuring the synopsis---not sure if these were desired, so feel free to edit or drop them completely. 

TLDR:
- added a front matter howto to easy-hugo-org-header users
- removed a false positive feature absence/conflict
- introduced some changes to the readme intro (at your discretion)

_(My very first PR on GitHub was on this repo and you kindly welcomed it even though I was a newcomer. That encouraged me to learn more and contribute to open-source projects that I use. So thank you for your time and dedication to this package. I was happy to see it featured in the Hugo documentation.)_

### CODE REVIEW

**Comment by ttybitnik** [comment]
Well, not sure if I like the logo centered anymore hahaha ¯\_(ツ)_/¯

(Let me know and I revert it if you want)

**Comment by masasam** [comment]
Hi @ttybitnik !
Thank you for PR!
I'm also happy to be included in hugo's documentation.
Please continue to help open source.
